packages/shared/styles/variables/_colors.scss

```diff
@@ -108,3 +108,34 @@ $colors-map: (
     'text': $c-white
   )
 );
+
+$icon-sizes : (
+    "xxs": 1rem,
+    "xs": 1.4rem,
+    "sm": 1.625rem,
+    "md": 1.8rem,
+    "lg": 2rem,
+    "xl": 2.2rem,
+    "xxl": 2.5rem,
+    "xl3": 2.8rem,
+    "xl4": 3.25rem
+);
+$icon-colors : (
+    "primary" : $c-green-primary,
```

Comment:
@przspa you think it's a problem?

Reply:
I'm not sure why @thiagoabreudev you moved it here :) you have some reason for it. I think now we should have icon variable in SfIcon.scss

Reply:
I'm sorry, I really got confused in this code. I was having trouble working with the icon colors in SfBottomNavigatio. That's why I had done it that way, but then I saw that everything I had done was extremely unnecessary and it wasn't working just because I didn't notice that the color of the icon in SfIcon is working with css vars.

I adjusted the code. Many thanks for the review @leomp12 and @przspa